plugins/javadoc/src/main/resources/views/class.korte

```diff
@@ -254,8 +262,10 @@
                             </a>
                                 <h3>Method Detail</h3>
                                 {% for method in methods.own %}
-                                <a name="{{ method.anchorLink }}">
-                                <ul class="blockListLast">
+                                <a id="{{ method.anchorLink }}">
+                                    <!--   -->
+                                </a>
+                                <ul class="blockList"> <!-- TODO: last item should have `blockListLast` class -->
```

Comment:
you can do it with 

```
{% if loop.last %} 
    "blockListLast" 
{% else %}
    "blockList"
{% endfor %}
```

https://korlibs.soywiz.com/korte/#for

Reply:
Great, thanks!